source/docs/articles/sites/cloudflare-cdn-and-dns.md

```diff
@@ -52,5 +52,28 @@ In Drupal, no modules are required to use CloudFlare, but there is an optional [
 
 **Railgun**: At this time, CloudFlare Railgun is not available on Pantheon.
 
-https://support.cloudflare.com/hc/en-us/articles/200169456-Why-is-JavaScript-or-jQuery-not-working-on-my-site-
+**Rocket Loader**: Has the potential to break some JavaScript and jQuery functions as a beta feature. [Read CloudFlare's support article](https://support.cloudflare.com/hc/en-us/articles/200169456-Why-is-JavaScript-or-jQuery-not-working-on-my-site-).
+
+
+## FAQs
+
+**Why isn't JavaScript or jQuery working on my site?**
+
+If your JS files are not loading, check Firebug to see if one of your links shows the following error:
+
+```
+MIME-Version: 1.0
+Content-Type: multipart/mixed; boundary="qA7D'DGwpMH,)x"
+
+--qA7D'DGwpMH,)x
+X-Cf-Url: http://www.domain.com/sites/all/modules/admin_menu/admin_menu.js?nvql2t
+X-Cf-Status: 403
+Content-Transfer-Encoding: binary
+X-Cf-Max-Age: 7200
+X-Cf-Origin-Status: 403
+X-Cf-Cache-Status: EXPIRED
+Content-Type: text/html; charset=UTF-8
+X-Cf-Error: true
+```
+
```

Comment:
@nataliejeremy should we add that the solution to this error is to disable the Rocket Loader feature in CloudFlare?